brave/browser-laptop
# Sorted the entire ledger before display.

Fix #12119
@bsclifton 
Submitter Checklist:


Test Plan:
1. Simulate payment history: npm run add-simulated-payment-history 50
2. Build from source: npm run watch
3. Run the program: npm start
4. Check the order of payment history: Go to brave setting, under Payments, click on "Your Payment History" and check that the order is from newest to oldest.

Reviewer Checklist:

Tests
